### ISSUE TITLE
Several patch releases contain breaking changes

### ISSUE THREAD

**Issue by mdenburger** [ava/use-http#267]
**Describe the bug**
Several patch releases now contain breaking API changes, which violates [semantic versioning](https://semver.org/).

**Expected behavior**
From version 1.0.0 onwards I would expect that every patch release is fully backwards compatible, and upgrading is a no-brainer. New features would increase the minor version (0.1.0, 0.2.0, etc). Any backwards incompatible change would lead to a new major version (2.0.0, 3.0.0, etc.).

Writing "THIS RELEASE CONTAINS BREAKING CHANGES" in the release notes is a big smell. Whether a release contains breaking changes should be communicated via the semantic version number.

**Comment by alex-cory** [comment]
Yeah, I jumped the gun on `v1.0` 😥. I thought it was ready. I will fix asap. My sincerest apologies.
